kvbc/src/direct_kv_db_adapter.cpp

```diff
@@ -364,13 +364,17 @@ KeyValuePair DBKeyManipulator::composedToSimple(KeyValuePair _p) {
   return KeyValuePair(key, _p.second);
 }
 
-DBAdapter::DBAdapter(std::shared_ptr<storage::IDBClient> db, std::unique_ptr<IDataKeyGenerator> keyGen, bool use_mdt)
+DBAdapter::DBAdapter(std::shared_ptr<storage::IDBClient> db,
+                     std::unique_ptr<IDataKeyGenerator> keyGen,
+                     bool use_mdt,
+                     bool save_kv_pairs_separately)
     : logger_{logging::getLogger("concord.kvbc.v1DirectKeyValue.DBAdapter")},
       db_(db),
       keyGen_{std::move(keyGen)},
       mdt_{use_mdt},
       lastBlockId_{fetchLatestBlockId()},
-      lastReachableBlockId_{fetchLastReachableBlockId()} {}
+      lastReachableBlockId_{fetchLastReachableBlockId()},
+      saveKvPairsSeparately_{save_kv_pairs_separately} {}
```

Comment:
Nit: I guess that's the existing naming in the file, but it seems inconsistent.

Reply:
Yes, for some reason function parameter names are with _, while members with camel case and trailing _. I'll leave it like this.